app/controllers/services_and_information_controller.rb

```diff
@@ -0,0 +1,18 @@
+class ServicesAndInformationController < ApplicationController
+  include ServicesAndInformationHelper
+
+  def index
+    base_path = "/government/organisations/#{params[:organisation_id]}/services-information"
+    content_item = Services.content_store.content_item!(base_path).to_hash
+    @navigation_helpers = GovukNavigationHelpers::NavigationHelper.new(content_item)
+    @organisation = content_item.dig("links", "parent", 0)
+    @grouped_links = parsed_grouped_links
```

Comment:
This is better as class you can instantiate, instead of a helper. That makes it clear where the method comes from.